pipelines/SpeciesTreeFromBusco/main.nf

```diff
@@ -117,7 +117,7 @@ process prepareGenome {
         path "processed/*", emit: proc_genome
     when: params.dir != ""
     script:
-        id = (genome =~ /(.+)\..+?$/)[0][1]
+        id = (genome =~ /(.+)\.fas?(.gz)?$/)[0][1]
```

Comment:
Could this be used to make the pattern more specific?
```suggestion
        id = (genome =~ /(.+)\.fas?(\.gz)?$/)[0][1]
```

Reply:
Yes, the specific version is better. I missed quoting the dot.

Reply:
No worries, I've missed the same thing before on at least one occasion.